internal/manifest/compose/parse.go

```diff
@@ -1,7 +1,23 @@
 package compose
 
 import (
-	"github.com/deref/exo/internal/providers/docker/compose"
+	"io"
+
+	"github.com/goccy/go-yaml"
 )
 
-var Parse = compose.Parse
+type Project struct {
```

Comment:
"Project" is apparently what Docker Compose now calls these things, tho it really means the merge of multiple compose files.